delivery_postlogistics/postlogistics/web_service.py

```diff
@@ -16,6 +16,9 @@
 
 from odoo import _, exceptions
 from odoo.exceptions import UserError
+from odoo.tools.translate import LazyTranslate
+
+_lt = LazyTranslate(__name__, default_lang="en_US")
```

Comment:
why is `LazyTranslate` needed? (I'm not familiar with its usage)

Reply:
When the default language is unavailable and a log with a traceback is generated, test cases fail. This issue arises because the translation system expects a valid language context, which may not always be present.
https://github.com/odoo/odoo/blob/18.0/odoo/tools/translate.py#L587

In this particular case, the `PostlogisticsWebService` class is not an Odoo class, which could explain why it lacks Odoo’s built-in translation functionality. In the Odoo codebase, LazyTranslate has been used in similar scenarios to avoid such issues:
https://github.com/odoo/odoo/blob/b2e564f07f6883111da4581bccfb0f45c7588063/odoo/tools/image.py#L88

Additionally, I was not familiar with self.env._, but it would face the same issue here since `self` does not have an `env` in this context.